plugins/Marketplace/Plugins/InvalidLicenses.php

```diff
@@ -223,22 +223,13 @@ public function setActivatedPluginNames($pluginNames)
         $this->activatedPluginNames = $pluginNames;
     }
 
-    protected function isPluginInstalled($pluginName)
-    {
-        if (in_array($pluginName, $this->activatedPluginNames)) {
-            return true;
-        }
-
-        return $this->pluginManager->isPluginInstalled($pluginName);
-    }
-
     protected function isPluginActivated($pluginName)
```

Comment:
small suggestion: could rename the method to, eg, `isPluginInActivatedPluginsList` to differentiate from `Manager::isPluginActivated`.